libr/util/sys.c

```diff
@@ -1143,7 +1143,21 @@ R_API bool r_sys_tts(const char *txt, bool bg) {
 R_API const char *r_sys_prefix(const char *pfx) {
 	static char prefix[1024] = {0};
 	if (!*prefix) {
+#if __WINDOWS__ && !CUTTER
+		char *path = r_sys_cmd_str ("where radare2", NULL, NULL);
+		if (path && *path) {
+			char *dir, *tmp = dir = r_file_dirname (path);
+			dir = r_file_dirname (tmp);
+			if (dir) {
+				r_str_ncpy (prefix, dir, sizeof (prefix));
+			}
+			free (dir);
+			free (tmp);
+		}
```

Comment:
What happens if `!path`?
On Windows, since (afaik) most of the people just copy paste the binaries, r2 is most of the time not in the `PATH`.

Reply:
with the installer script radare is added to path, and if radare is not in path, I guess it should fall back to searching on cwd